wdl/src/main/scala/wdl/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -176,6 +176,22 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WomValue] {
     extractArguments flatMap arrayLength
   }
 
+  def flatten(params: Seq[Try[WomValue]]): Try[WomValue] = {
+    def getFlatValues(v: WomValue): List[WomValue] = v match {
+      case WomArray(_, values) => values.toList
+      case other => List(other)
+    }
+
+    val arg:Try[WomValue] = extractSingleArgument("flatten", params)
+    arg flatMap {
+      case WomArray(WomArrayType(WomArrayType(elemType)), arrayValues) =>
+        val ll:List[List[WomValue]] = arrayValues.toList.map(getFlatValues)
```

Comment:
I feel like this map followed by flatten could be structured into a `flatMap`, but only do it if it looks easy